### ISSUE TITLE
The F6 part navigation does not work if no editor is open

### ISSUE THREAD

**Issue by lpintes** [microsoft/vscode#115677]
Issue Type: <b>Bug</b>

Open vscode and try to press F6. Previously it cycled between various part of the interface. Now it is stuck.

VS Code version: Code - Insiders 1.53.0-insider (8d779a4fadaadec06b44b8875a1d3c4adba8e380, 2021-02-02T16:34:18.896Z)
OS version: Windows_NT x64 10.0.19042

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-3720QM CPU @ 2.60GHz (8 x 2594)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|7.58GB (2.82GB free)|
|Process Argv|. --crash-reporter-id 3229a6ba-f91c-4af0-945c-d57b1e4677fb|
|Screen Reader|yes|
|VM|0%|
</details><details><summary>Extensions (36)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-sqlite|ale|0.11.1
npm-intellisense|chr|1.3.1
vscode-markdownlint|Dav|0.38.0
jshint|dba|0.11.0
vscode-eslint|dba|2.1.14
javascript-ejs-support|Dig|1.3.1
gitlens|eam|11.2.1
EditorConfig|Edi|0.16.4
vscode-npm-script|eg2|0.3.13
elm-ls-vscode|elm|2.0.2
vscode-pull-request-github|Git|0.22.0
go|gol|0.22.0
beautify|Hoo|1.5.0
rest-client|hum|0.24.4
ilspy-vscode|ics|0.9.0
csharpextensions|jch|1.3.1
docomment|k--|0.1.20
goto-next-previous-member|mis|0.0.5
vscode-docker|ms-|1.9.1
csharp|ms-|1.23.8
python|ms-|2021.1.502429796
jupyter|ms-|2020.12.414227025
remote-containers|ms-|0.158.0
remote-wsl|ms-|0.53.2
cpptools|ms-|1.2.0
debugger-for-chrome|msj|4.12.12
java|red|0.74.0
vscode-xml|red|0.14.0
sonarlint-vscode|Son|1.19.0
vscodeintellicode|Vis|1.2.11
vscode-java-debug|vsc|0.31.0
vscode-java-dependency|vsc|0.17.0
vscode-java-pack|vsc|0.12.1
vscode-java-test|vsc|0.28.0
vscode-maven|vsc|0.27.1
markdown-all-in-one|yzh|3.4.0

</details><details>
<summary>A/B Experiments</summary>

```
vsliv695:30137379
vsins829:30139715
vsliv368cf:30146710
vsreu685:30147344
openlogontheside:30221882
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vswsl492:30208929
wsl2prompt:30219162
vstry244:30244315
pythonvsdeb440:30224570
unusedpromptcf:30219165
folderexplorer:30219166
openfilemenu:30219168
pythonvsded773:30223139
pythonvspyt600cf:30251589
core-portspanelcf:30233468
coreuntitledfile:30249963

```

</details>

**Comment by isidorn** [comment]
Thanks for filling this issue. This works for me with latest insiders.
Can you reproduce using Visual Studio Code Insiders https://code.visualstudio.com/insiders/

**Comment by lpintes** [comment]
Hello,

I am using the latest insiders updated today and I still think it 
doesn't work like before.

Try to open an empty vscode, also close the untitled-1 document that 
opens automatically. Then try F6 and Shift+F6. It works for a very first 
time, but is stuck after.

Dňa 3. 2. 2021 o 10:33 Isidor Nikolic napísal(a):

**Comment by isidorn** [comment]
Okey, I can only reproduce if no editors are open. Looks like a valid bug.

**Comment by isidorn** [comment]
Thanks again for filling this issue!
The fix should be in vscode insiders from Friday, so try it out and let us know how it goes.
